hera_cal/abscal_funcs.py

```diff
@@ -901,10 +1077,9 @@ def gains2calfits(calfits_fname, abscal_gains, freq_array, time_array, pol_array
 
     Parameters:
     -----------
-    calfits_fname : string, path and filename to output calfits file
+    calfits_fname : type=string, path and filename to output calfits file
 
-    abscal_gains : dictionary, antenna integer as key, ndarray complex gain
-        as values with shape (Ntimes, Nfreqs, Npols)
+    abscal_gains : type=dictionary, (ant, pol) as key, (Ntimes, Nfreqs) complex ndarray as value
```

Comment:
gain_convention and overwrite should be in the list of parameters

Reply:
yup